#request
geometry_msgs/Pose T_0_EE_old
---
#response
geometry_msgs/Pose T_0_EE_new
